requirements.txt

```diff
@@ -23,7 +23,7 @@ oset==0.1.3
 pandas==0.22.0
 pbr==3.1.1
 progressbar2==3.6.2
-pybtex==0.21
+pybtex==0.24
```

Comment:
version 0.21 won't build with the latest setup tools due to `use_2to3` being a deprecated option.

Reply:
Why is this package needed and where is it used? The current version removed a lot of requirements that aren't needed.